src/sentry/incidents/action_handlers.py

```diff
@@ -47,10 +48,12 @@ def resolve(self, metric_value: int | float, new_status: IncidentStatus):
 
 class DefaultActionHandler(ActionHandler):
     def fire(self, metric_value: int | float, new_status: IncidentStatus):
-        self.send_alert(metric_value, new_status)
+        if not RuleSnooze.objects.filter(alert_rule=self.incident.alert_rule).exists():
+            self.send_alert(metric_value, new_status)
 
     def resolve(self, metric_value: int | float, new_status: IncidentStatus):
-        self.send_alert(metric_value, new_status)
+        if not RuleSnooze.objects.filter(alert_rule=self.incident.alert_rule).exists():
+            self.send_alert(metric_value, new_status)
```

Comment:
Actually should this be filtered to `RuleSnooze` rows that are only not for specific users?

Reply:
oh shit yeah